### PR TITLE
Addition of database dev setup.

### PR DESCRIPTION
**Pull request by AdamWaldie**
Does not include DB intergration.

### CODE REVIEW

File: tarkas/arma3/development/development2_Dev.json [hunk 1]
```diff
@@ -71,7 +71,7 @@
             },
             {
                 "app": "751965892",
-                "name": "@ACRE",
+                "name": "@ACRE2",
```
Comment (Jarvvski) [inline tarkas/arma3/development/development2_Dev.json:74]: Does this really matter?

It's not really adding anything

Comment (AdamWaldie) [reply tarkas/arma3/development/development2_Dev.json:74]: Nope, doesnt matter, for some reason it just isn't labelled ACRE in my fork. Might need to update mine.

Comment (Jarvvski) [reply tarkas/arma3/development/development2_Dev.json:74]: can we remove it from this PR then?

Comment (AdamWaldie) [reply tarkas/arma3/development/development2_Dev.json:74]: Yes indeed

**Pull request merged**